zbsprimarystorage/zbsprimarystorage/zbsagent.py

```
@@ -568,7 +568,7 @@ def clean_nbd(self, req):
         rsp = AgentResponse()
 
         fullname = "qemu-nbd -D cbd2nbd.%d -f raw -p %d" % (cmd.port, cmd.port)
-        linux.kill_process_by_fullname(fullname, 9)
+        linux.kill_process_by_fullname(fullname, 15)
```

Ruff (0.13.1)

`linux` may be undefined, or defined from star imports

(F405)